### DISCUSSION TITLE
Remove old drive info from Grafana

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#9295] [Unsupported Versions]
Since my initial install of SO (in a VM) I had changed my hard drive type to make use of fstrim and reclaim unused blocks in my virtual hard drive.  Hence, my drives changed from /dev/vda to /dev/sda.  Grafana has kept the old drive info, which is now useless and also Ihad a USB hard drive plugged in at one point for somthing else and Granfana made graphs for it also.

Since these graphs are empty and useless, just taking up space on my screen, is there a way to permanently remove or hide these?  I think I tried hiding them at one point, but they reappeared.  Thanks.

**Comment by dougburks** [comment]
I suspect that the old data will age out automatically within a few weeks. If you don't want to wait, you could try `so-influxdb-clean`:
https://docs.securityonion.net/en/2.3/grafana.html#grafana-changes-in-security-onion-2-3-60

**Comment by DebianGuru** [comment]
That did it!  Thanks Doug.
